ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/ContractStates.kt

```diff
@@ -0,0 +1,86 @@
+package net.corda.v5.ledger.utxo
+
+import net.corda.v5.base.annotations.CordaSerializable
+import java.security.PublicKey
+import java.util.*
+import kotlin.reflect.KClass
+
+/**
+ * Indicates the [Contract] that the current state belongs to.
+ */
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.CLASS)
+annotation class BelongsToContract(val value: KClass<out Contract>)
+
+/**
+ * Indicates the URI that references the legal prose associated with the current [Contract].
+ */
+@Target(AnnotationTarget.CLASS)
+@MustBeDocumented
+annotation class LegalProseReference(val value: String)
```

Comment:
Delete please

Reply:
@mnesbit Why don't we don't want references for legal prose anymore?

Reply:
Legal prose argument is a black hole basically. We have people who won't be able to upgrade from C4 to C5, because they have written that their code forms their legal master contract and thus upgrading isn't just a software upgrade, but in fact a renegotiation of legal terms! Plus what is the version relationship of an annotation if you do want to upgrade the text? I would definitely delete until we have a customer who wants a mechanism for this.